packages/amplify-category-geo/src/__tests__/service-utils/validateGeoJSONObj.test.ts

```diff
@@ -0,0 +1,32 @@
+import path from 'path';
+import { readFileSync } from 'fs-extra';
+import { FeatureCollection, IdentifierOption } from '../../service-utils/importParams';
+import { validateGeoJSONObj } from '../../service-utils/validateGeoJSONObj';
+
+const assetRoot = path.resolve(path.join(__dirname, '..', 'assets'));
+
+const loadGeoJSONObj = (geoJSONFileName: string): FeatureCollection => JSON.parse(readFileSync(path.join(assetRoot, geoJSONFileName), 'utf-8')) as FeatureCollection;
+
+describe('Geo JSON file validation test', () => {
+  it('should not throw error for a valid Geo JSON file', () => {
+    expect(() => validateGeoJSONObj(loadGeoJSONObj('validGeofence.json'))).not.toThrow();
+  });
+  it('should throw error for a invalid Geo JSON file with first and last position not identical', () => {
+    expect(() => validateGeoJSONObj(loadGeoJSONObj('invalidGeofence.notSameFirstAndLast.json'))).toThrowErrorMatchingSnapshot();
```

Comment:
For all these tests with smaller snapshots, we should use `toThrowErrorMatchingInlineSnapshot`